include/SDL3/SDL_video.h

```diff
@@ -1675,6 +1675,20 @@ extern DECLSPEC int SDLCALL SDL_SetWindowModalFor(SDL_Window *modal_window, SDL_
  */
 extern DECLSPEC int SDLCALL SDL_SetWindowInputFocus(SDL_Window *window);
 
+/**
+ * Display the system-level window menu.
+ *
+ * \param window the window for which the menu will be displayed
+ * \param x the x coordinate of the menu, relative to the origin of the window
```

Comment:
Is this the origin of the client area or the window decorations? Is the origin top left or bottom left?